pkg/controller/volume/expand/cache/volume_resize_map.go

```diff
@@ -160,6 +163,19 @@ func (resizeMap *volumeResizeMap) MarkAsResized(pvcr *PVCWithResizeRequest, newS
 	return nil
 }
 
+// MarkForFSResize marks pvc with condition that indicates a fs resize is pending
+func (resizeMap *volumeResizeMap) MarkForFSResize(pvcr *PVCWithResizeRequest) error {
+	pvcCondition := v1.PersistentVolumeClaimCondition{
+		Type:               v1.PersistentVolumeClaimFileSystemResizePending,
+		Status:             v1.ConditionTrue,
+		LastTransitionTime: metav1.Now(),
+		Message:            "Restart pod that is using this claim to finish File system resize on node",
```

Comment:
IMO, it should say why the condition is set. "Waiting for user to (re-)start a pod to finish resize of the volume" or something in this direction?

Reply:
fixed